src/metabase/api/user.clj

```diff
@@ -33,7 +33,7 @@
    [toucan2.core :as t2]))
 
 (defsetting user-visibility
-  (deferred-tru "Determines what other users non-admin users are able to see. Possible values are :all , :group, or :none.")
+  (deferred-tru "Note: Sandboxed users will never see suggestions.")
```

Comment:
Please ask Maz to check if this copy change important enough to merge it now and send to translators

Reply:
Thanks for bringing this up, I started a thread https://metaboat.slack.com/archives/C864UT5CZ/p1689957867557039

Reply:
Got the okay from Maz to merge this, we just need to let him know when the feature flags branch gets merged into the release branch so he can pull these changes into the translation editor